native_client/README.rst

```diff
@@ -197,3 +198,26 @@ And (ARM64):
 
    cd ../DeepSpeech/native_client
    $ANDROID_NDK_HOME/ndk-build APP_PLATFORM=android-21 APP_BUILD_SCRIPT=$(pwd)/Android.mk NDK_PROJECT_PATH=$(pwd) APP_STL=c++_shared TFDIR=$(pwd)/../tensorflow/ TARGET_ARCH_ABI=arm64-v8a
+
+Delegation API
+^^^^^^^^^^^^^^
+
+TensorFlow Lite supports Delegate API to offload some computation from the main
+CPU. Please refer to `TensorFlow's documentation
+<https://www.tensorflow.org/lite/performance/delegates>`_ for details.
+
+To ease with experimentations, we have enabled some of those delegations on our
+Android builds: * GPU, to leverage OpenGL capabilities * NNAPI, the Android API
+to leverage GPU / DSP / NPU * Hexagon, the Qualcomm-specific DSP
```

Comment:
Looks like something messed up the newlines/lists here.

Reply:
Too Quick in rewriting the list in vim 😢

Reply:
Too Quick in rewriting the list in vim 😢